### PR TITLE
Linting Tools and CI

### PR DESCRIPTION
**Pull request by murilovmachado**
Jira: https://bugs.benetech.org/browse/CHI-50

This PR:
- Adds `eslint` and `prettier`
- Fixes all linting issues
- Gets rid of all vunarabilities updating the following dependencies:
  - sequelize
  - jest
- Setup CI
- Adds CI status badge to `README`

@nickhurlburt Let's move the creation of tests to another card, please.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  "extends": ["airbnb-base", "prettier"],
```
Comment (nickhurlburt) [inline .eslintrc.js]: Out of curiosity -- why Airbnb?

File: .github/workflows/hrm-ci.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
```
Comment (nickhurlburt) [inline .github/workflows/hrm-ci.yml]: Great.  Side note -- I've just upgraded our AWS staging env to Node 12.16.1.  It had been on Node 10.  If that all goes well we can also upgrade production.

File: app.js [hunk 3]
```diff
@@ -2,169 +2,143 @@ const createError = require('http-errors');
...
 const host = process.env.RDS_HOSTNAME || 'localhost';
 const user = process.env.RDS_USERNAME || 'hrm';
 const pass = process.env.RDS_PASSWORD || '';
-const port = process.env.RDS_PORT || '5432';
 const apiKey = process.env.API_KEY;
 
-const Op = Sequelize.Op;
-const version = "0.3.0";
+const { Op } = Sequelize;
+const version = '0.3.0';
```
Comment (nickhurlburt) [inline app.js]: Ah... I roughly try to keep this in sync with flex-plugins whenever we change code here... so let's make this 0.3.6 now.

**Comment by nickhurlburt** [comment]
@murilovmachado also yes, we can make testing a separate card.

File: .eslintrc.js (hunk 1 above)
Comment (murilovmachado) [reply .eslintrc.js]: It's the most used lint configuration in Javascript projects, with very good documentation.

**Pull request merged**